internal/provider/exemption_resource_test.go

```diff
@@ -0,0 +1,57 @@
+package provider
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestAccExemptionResource(t *testing.T) {
+	CheckAccTestsEnabled(t)
+
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Create + Read.
+			{
+				Config: providerConfig + `
+resource "adaptive-metrics_exemption" "test" {
+	metric = "test_tf_metric"
```

Comment:
Is the metric here a unique key? If it is, I'd switch to a random value so you don't get conflicts between runs

Reply:
Nope, not unique but this makes me realize it's a problem for the rule resource - thanks for pointing this out!